test_unified_integration.py

```
@@ -0,0 +1,59 @@
+"""
+Test script to verify unified ingestion pipeline integration.
+"""
+import asyncio
+import sys
+import os
+sys.path.append(os.path.dirname(__file__))
+
+from unified_ingestion.pipeline import UnifiedIngestionPipeline
+from unified_ingestion.data_models import ContentType, IngestionConfig
+
+async def test_unified_pipeline():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_unified_pipeline`

Add return type annotation: `bool | None`

(ANN201)